components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/ConfigurationManager.java

```diff
@@ -100,6 +100,19 @@ Resources getTenantResources(String tenantDomain, Condition searchCondition) thr
      */
     Resources getResources() throws ConfigurationManagementException;
 
+    /**
+     * Get all the resources belongs to the given {@link ResourceType} for a given tenant ID.
+
+     * @param tenantId         The ID of the tenant.
+     * @param resourceTypeName {@link ResourceType} object name.
+     * @return 200 ok. @link Resources} object with all the resources of the given resource type name.
```

Comment:
200 ok?

Reply:
Yes I used the same comment as the overloaded method.

Reply:
Removed